pages/views/frontend.py

```diff
@@ -56,63 +56,32 @@ def frontend(request):
         request.session.set_expiry(0)
         
         # grab client cert form the request create user hash, ignore if no cert info is found in request
-        
         try:
             certInfo = getCert(request)
+            cftsUser = getOrCreateUser(request, certInfo)
+
+            if cftsUser == None or not request.user.is_authenticated:
+                return redirect("/login")
+            elif cftsUser.update_info == True:
+                return redirect("/user-info")
+            elif cftsUser.banned == True:
```

Comment:
- user ban status should not be a part of an if else statement, line 67
- remove elif statement for cftsUser.banned and replace it with a call to checkBan()

Reply:
Done